index.html

```diff
@@ -102,9 +102,10 @@
 doing irreparable harm to individual autonomy online.
 </section>
 
-<section id="sotd">
-This is a draft TAG finding and does not yet represent TAG consensus.
-</section>
+<p id=sotd>
```

Comment:
This ought to stay a section. Respec injects an `<h2>`, which causes the `<p>` to close itself.

```suggestion
<section id=sotd>
```